keyboards/qwertlekeys/calice/config.h

```diff
@@ -0,0 +1,54 @@
+ /* Copyright 2021 Joah Nelson (Jels)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation,   either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor paramater */
+#define VENDOR_ID       0x716b     //qk (qwertlekeys)
+#define PRODUCT_ID      0x0001     //Calice
+#define DEVICE_VER      0x0001      
+#define MANUFACTURER    QwertleKeys //pcb by jels
+#define PRODUCT         Calice
+
```

Comment:
```suggestion
```

Reply:
Is this the new way of declaring device info?

Reply:
Yes.